content-webapp/src/main/webapp/schedule-news-drawer/components/ExoScheduleNewsDrawer.vue

```diff
@@ -21,7 +21,7 @@
 <template>
   <div>
     <v-overlay
-      z-index="1000"
+      z-index="3000"
```

Comment:
do this change have a impact on old news composer ?

Reply:
Nope